bigchaindb/util.py

```diff
@@ -324,10 +326,11 @@ def sign_tx(transaction, signing_keys):
 
     tx = copy.deepcopy(transaction)
 
+    bigchain = bigchain if bigchain is not None else bigchaindb.Bigchain()
```

Comment:
I guess that would work in certain situations but it seems risky and not fun to debug
For instance
```python
bigchain = bigchain or bigchaindb.Bigchain()
```
is different from
```python
bigchain = bigchaindb.Bigchain() or bigchain
```